app/src/main/res/drawable/shape_status_ok_more_info.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners
+        android:bottomLeftRadius="4dp"
+        android:bottomRightRadius="4dp" />
+
+    <solid android:color="@color/status_ok_more_info_bg" />
+
+    <size
+        android:width="100dp"
+        android:height="100dp" />
+</shape>
```

Comment:
I guess you missed this one https://github.com/covilights/AndroidCoviLights/pull/21#discussion_r413190270
please also rename this file to -> `shape_status_ok_more_info_bg.xml`